python/lsst/analysis/tools/actions/plot/barPlots.py

```diff
@@ -88,22 +92,22 @@ def makePlot(
         plotInfo : `dict`
             An optional dictionary of information about the data being
             plotted with keys:
-                `"run"`
-                    Output run for the plots (`str`).
-                `"tractTableType"`
-                    Table from which results are taken (`str`).
-                `"plotName"`
-                    Output plot name (`str`)
-                `"SN"`
-                    The global signal-to-noise data threshold (`float`)
-                `"skymap"`
-                    The type of skymap used for the data (`str`).
-                `"tract"`
-                    The tract that the data comes from (`int`).
-                `"bands"`
-                    The bands used for this data (`str` or `list`).
-                `"visit"`
-                    The visit that the data comes from (`int`)
+            `"run"`
+            Output run for the plots (`str`).
+            `"tractTableType"`
+            Table from which results are taken (`str`).
+            `"plotName"`
+            Output plot name (`str`)
+            `"SN"`
+            The global signal-to-noise data threshold (`float`)
+            `"skymap"`
+            The type of skymap used for the data (`str`).
+            `"tract"`
+            The tract that the data comes from (`int`).
+            `"bands"`
+            The bands used for this data (`str` or `list`).
+            `"visit"`
+            The visit that the data comes from (`int`)
```

Comment:
To get a dictionary list, start with a blank line then indent each description relative to its term:

```suggestion
            `"run"`
                Output run for the plots (`str`).
            `"tractTableType"`
                Table from which results are taken (`str`).
            `"plotName"`
                Output plot name (`str`)
            `"SN"`
                The global signal-to-noise data threshold (`float`)
            `"skymap"`
                The type of skymap used for the data (`str`).
            `"tract"`
                The tract that the data comes from (`int`).
            `"bands"`
                The bands used for this data (`str` or `list`).
            `"visit"`
                The visit that the data comes from (`int`)
```